src/validators/user-request-validator.js

```diff
@@ -0,0 +1,9 @@
+const Joi = require('@hapi/joi');
+
+const userRequestSchema = Joi.object({
+    user_id: Joi.number().required(),
+    user_name: Joi.string().required(),
```

Comment:
I believe it would be better to add min() validation to prevent user_name from being an empty string

Reply:
hmm, yeah but if `user_name` like first name we shouldn't put any limitations, so guess we can put min(3) (hope there're no name less than 3)